src/Generator/Generators/CSharp/CSharpTextTemplate.cs

```diff
@@ -353,6 +353,12 @@ public void GenerateInlineSummary(RawComment comment)
 
         public void GenerateClass(Class @class)
         {
+            if(@class.DeclaredStruct)
+            {
+                WriteLine("//Declared Struct");
```

Comment:
Sorry for this one, I forgot to erase it. I used it just to test if 'DeclaredStruct' was properly initialised.